src/execution_plan/optimizations/traverse_order.h

```diff
@@ -17,6 +17,7 @@ typedef enum {
 
 /* Reorders exps such that exp[i] is the ith expression to evaluate. */
 void orderExpressions(
+	QueryGraph *qg,                 // QueryGraph containing expression entity data.
```

Comment:
I think `qg` should be retrieved from TLS just so we can keep  `orderExpressions` signature shorter and cleaner.